### PR TITLE
Add run vae apps tests

### PR DESCRIPTION
**Pull request by wngzhiqi**
Add run vae apps tests to enable PR validation.

### CODE REVIEW

File: apps/variational_autoencoder/readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+3. [Using Variational Autoencoder and Deep Feature Loss to Generate Human Faces](./using_variational_autoencoder_and_deep_feature_loss_to_generate_faces.ipynb)
+   This notebook is similar to the previous one but use a more sophisticated loss function to generate
+   more vivid images at the cost of more training time.
+
+To run this example, you should 
+* Download Zoo and build it.
+* Set the environment variable `ZOO_HOME` to `/your_zoo_directory/dist` 
+* Run the bash command to start the notebook.
+```Bash
+${ZOO_HOME}/scripts/jupyter-with-zoo.sh \
```
Comment (yangw1234) [inline apps/variational_autoencoder/readme.md]: You may want to check this scripts works with latest master

File: apps/variational_autoencoder/using_variational_autoencoder_to_generate_faces.ipynb [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+    "import scipy.misc\n",
+    "import numpy as np\n",
+    "\n",
+    "from utils import *\n",
+    "\n",
+    "image_size = 148\n",
+    "Z_DIM = 128\n",
+    "ENCODER_FILTER_NUM = 32\n",
+    "\n",
+    "#download the data with download_dataset.sh\n",
```
Comment (yangw1234) [inline apps/variational_autoencoder/using_variational_autoencoder_to_generate_faces.ipynb]: delete this comment?

File: apps/variational_autoencoder/using_variational_autoencoder_and_deep_feature_loss_to_generate_faces.ipynb [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+    "import numpy as np\n",
+    "from utils import *\n",
+    "\n",
+    "image_size = 148\n",
+    "Z_DIM = 100\n",
+    "ENCODER_FILTER_NUM = 32\n",
+    "\n",
+    "# we use the vgg16 model, it should work on other popular CNN models\n",
+    "# You can download them from download_model.sh\n",
+    "# And download the datasets from download_dataset.sh\n",
```
[the remark this answers is not in the document]
Comment (wngzhiqi) [reply apps/variational_autoencoder/using_variational_autoencoder_and_deep_feature_loss_to_generate_faces.ipynb]: Already delete the comments.

File: apps/variational_autoencoder/readme.md (hunk 1 above)
Comment (wngzhiqi) [reply apps/variational_autoencoder/readme.md]: Change ZOO_HOME to ANALYTICS_ZOO_HOME

**Comment by wngzhiqi** [comment]
Jenkins apps test passed
http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Python-AppTests/34/console

**Comment by yangw1234** [comment]
@glorysdj can you help review the changes to apps/run-app-tests.sh

**Pull request merged**